docs/public-api.asciidoc

```diff
@@ -10,6 +10,58 @@ customize and manually create spans and transactions,
 as well as track errors.
 
 
+[float]
+[[api-initialization]]
+== Initialization
+
+The API does not require explicit Agent initialization - agent initialization is optional. The `Elastic.Apm.Agent.IsConfigured` property lets you check whether the agent is already initialized.
+
+[float]
+[[implicit-initialization]]
+=== Implicit agent initialization
+
+If you don't explicitly initialize the agent, it will be started with a default configuration. This means the agent will read <<configuration, configuration settings>> from environment variables. In case you don't set an environment variable, the Agent will use the default value (in case of `ServerUrls` this is `http://localhost:8200`).
```

Comment:
```suggestion
If you don't explicitly initialize the agent, it will be started with a default configuration. This means the Agent will read <<configuration,configuration settings>> from environment variables. If you don't set an environment variable, the Agent will use the default value. For example, the `ServerUrls` default is `http://localhost:8200`.
```